examples/plot_simulate_somato.py

```diff
@@ -82,5 +82,5 @@
 import matplotlib.pyplot as plt
 fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6))
 dpl[0].plot(ax=axes[0], show=False)
-net.plot_input(ax=axes[1])
+net.spikes.plot_input(ax=axes[1])
```

Comment:
I think we should just create `spikes` object from the constructor `Spikes` with mock data and test it. That way we can test all the different corner cases fast.

Reply:
Not sure I understand this comment. That makes sense for the tests, but for the above example wouldn't you want to use the data that was just simulated?

Reply:
well I was thinking that to test the various errors, you need to create a spikes object by doing `Spikes(...)`. But I realized now that we don't really test anything about what the spikes object contains. Your tests look fine to me.